src/ray/gcs/subscription_notification.h

```diff
@@ -0,0 +1,53 @@
+#ifndef RAY_GCS_NOTIFICATION_DEF_H
+#define RAY_GCS_NOTIFICATION_DEF_H
```

Comment:
usually this follows the name of the file

Reply:
This class already removed, please refresh.